src/gpu/generic/sycl/sycl_primitive_conf.hpp

```diff
@@ -415,6 +415,17 @@ struct sycl_pooling_bwd_conf_t : public sycl_pooling_base_conf_t {
     xpu::sycl::md_t diff_dst_md;
 };
 
+struct sycl_simple_reduction_conf_t {
+    dnnl_alg_kind_t alg = dnnl_alg_kind_undef;
+    xpu::sycl::md_t src_md;
+    xpu::sycl::md_t dst_md;
+    float p;
+    float eps;
+    sycl_post_ops_t post_ops;
+    dim_t reduce_dims[xpu::sycl::md_t::max_dims];
+    int reduce_size = 1;
```

Comment:
Any reason only this is initialised and not the floats?

Reply:
If you are referring to `p` and `eps`, those are set in `init_conf` to the values provided by the primitive desc, so they don't need to be explicitly initialised. `reduce_size` is used to accumulate the number of elements that are being reduced so it's convenient to initialise to 1 here so that it can directly be used in the loop.